[runtime-security] Generate SECL documentation (#8702)
DataDog/datadog-agent: docs/cloud-workload-security/scripts/templates/agent_expressions.md

---
title: Agent Expressions
kind: documentation
description: "Agent expression attributes and operators for Cloud Workload Security Rules"
disable_edit: true
further_reading:
- link: "/security_platform/cloud_workload_security/getting_started/"
  tag: "Documentation"
  text: "Get started with Datadog Cloud Workload Security"
---

## Agent expression syntax

> Rules for Cloud Workload Security (CWS) are first evaluated in the Datadog Agent to decide what system activity to collect. This portion of a CWS rule is called the Agent expression. Agent expressions use Datadog's Security Language (SECL). The standard format of a SECL expression is as follows:

Reviewer: ```suggestion
Cloud Workload Security (CWS) first evaluates activity within the Datadog Agent against agent expressions to decide what activity to collect. Agent expressions use Datadog's Security Language (SECL). The standard format of a SECL expression is as follows:
```

> Rules for Cloud Workload Security (CWS) are first evaluated in the Datadog Agent to decide what system activity to collect. This portion of a CWS rule is called the Agent expression. Agent expressions use Datadog's Security Language (SECL). The standard format of a SECL expression is as follows:

Reviewer 2: ```suggestion
Cloud Workload Security (CWS) first evaluates activity within the Datadog Agent against Agent expressions to decide what activity to collect. This portion of a CWS rule is called the Agent expression. Agent expressions use Datadog's Security Language (SECL). The standard format of a SECL expression is as follows:
```

Reviewer 2: Combined both edits
